book/src/reliable-vote-transmission.md

```diff
@@ -0,0 +1,133 @@
+# Reliable Vote Transmission
+
+Validator votes are messages that have a critical function for
+consensus and continuous operation of the network. Therefore it is
+critical that they are reliably delivered and encoded into the
+ledger.
+
+## Challenges
+
+1. Leader rotation is triggered by PoH, which is clock with high
+drift.  So many nodes are likely to have an incorrect view if the
+next leader is active in realtime or not.
+
+2. The next leader maybe easily be flooded.  Thus a DDOS would not
+only prevent delivery of regular transactions, but also consensus
+messages.
+
+3. Udp is unreliable, and our asynchronous protocol requires any
+message that is transmitted to be retransmitted until it is observed
+in the ledger.  Potentially creating a DDOS against the leader with
+a large number of validators.  Worst case flood would be Number of
```

Comment:
Can you use either a notation that uses Rust conventions (`num_nodes * num_retransmits`) or English ("product of the number of nodes and number of retransmits")?